crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -323,10 +323,11 @@ where
         _: &StateWatcher,
         _: Self::TaskParams,
     ) -> anyhow::Result<Self::Task> {
-        let mut db_tx = self.database.transaction();
-        let total_tx_count = db_tx.increase_tx_count(0).unwrap_or_default();
-        db_tx.commit()?;
-        graphql_metrics().total_txs_count.set(total_tx_count as i64);
+        {
+            let db_tx = self.database.transaction();
```

Comment:
No, we don't, but it will require changing of constraints for the port=D I'm okay with using transaction for now to just fix the issue and release a new `fuel-core` with the fix=)